src/manifest/manifest_schema.ts

```diff
@@ -207,10 +207,13 @@ export type ManifestFunction = ISlackFunctionDefinition<any, any, any, any>;
 
 export type ManifestFunctionsSchema = { [key: string]: ManifestFunctionSchema };
 
+export type ManifestFunctionType = "API" | "app" | undefined;
+
 export type ManifestFunctionSchema = {
+  type?: ManifestFunctionType;
   title?: string;
   description?: string;
-  source_file: string;
+  source_file?: string;
```

Comment:
`ManifestFunctionSchema` is the shape of the manifest's functions object. Since we are focused on the consumption of Connectors at the moment, we don't need to support them being added to the manifest yet. I'm inclined to not change this yet so that devs aren't able to register Connectors to their own manifests before we fully support it.

Reply:
Should we set `source_file=""` for connectors or create a new `type`?

Reply:
Do we need to if connectors don't have an export?

Reply:
Seems like we may need to implement `export` in `ConnectorFunctionDefinition` I expand on this in [this comment](https://github.com/slackapi/deno-slack-sdk/pull/176#discussion_r1214698723)

Reply:
We would have to add the following where ever `export` or `registerParameterTypes` is used

```ts
if (fn.export) {
}

if (fn.registerParameterTypes) {
}
```

They seem to only be used in 2 places